### DISCUSSION TITLE
Upgrading rke2 version 1.18 via tarball method

### DISCUSSION THREAD

**Discussion by EnzisZero** [rancher/rke2#3597] [General]
Hello!

I'm trying to upgrade our instance of rke2 version 1.18 to the latest (1.23.14+rke2r1). We're using the Air-Gap installation method via:
https://docs.rke2.io/install/airgap#tarball-method

What I'm wondering; is our version too old to work with this upgrade method? Do we need to do a fresh install instead? We're running into some awkward issues and will keep at it if the upgrade is expected to work

**Comment by brandond** [comment]
It should work, but you definitely can't go directly from 1.18 to 1.24. You should upgrade to the latest 1.18 patch release, then the latest 1.19, then 1.20, and so on. See https://kubernetes.io/releases/version-skew-policy/#supported-component-upgrade-order for more information.

I will also note that 1.18 was pretty early in the development of RKE2, so you may run into various bugs or bits of behavior that have changed significantly between then and now.

**Reply by EnzisZero** [reply]
Thanks for the fast reply! You are awesome :). We will update our process to do it properly.
